woocommerce/wc-admin
# Update coupon search to allow searching by code

Fixes #1611 

Searches coupons by code to avoid mismatched results by description and blocking result sets > 10.

### Before
<img width="446" alt="Screen Shot 2019-03-14 at 3 17 26 PM" src="https://user-images.githubusercontent.com/10561050/54338244-5223c900-466c-11e9-8709-2d8fed465e86.png">

### After
<img width="475" alt="Screen Shot 2019-03-14 at 3 18 09 PM" src="https://user-images.githubusercontent.com/10561050/54338280-67005c80-466c-11e9-8ca8-2db102d98c9c.png">

### Detailed test instructions:

1.  Add a coupon with a keyword in the description, but not in the coupon code (title).
2.  Search for that keyword under the coupons table.
3.  Check that the result is not returned.
4.  Check that the results in the network request for the autocompleter also do not return that coupon.
